app/helpers/search_helpers.py

```diff
@@ -1,6 +1,18 @@
 from math import ceil
 
 
+def get_lot_label(lot_param_value):
```

Comment:
Should this end up in utils at some point? Can see buyer/supplier apps utilising it.